koku/api/settings/test/test_views.py

```diff
@@ -99,8 +99,9 @@ def test_post_settings_tag_enabled(self):
             ocp_list = [f"openshift-{tag}" for tag in tags]
             keys_list = aws_list + ocp_list
             max_idx = len(keys_list)
+
+        enabled_tags = list(set(random.choices(keys_list, k=random.randint(0, max_idx))))
```

Comment:
I don't think it will matter much for this test, but `random` uses a pseudo-random generator while `secrets` provide more secure (actually random) randomness.
```suggestion
        enabled_tags = list(set(secrets.choice(keys_list) for _ in range(secrets.randbelow(max_idx))))
```

Higher level, this is changing the test from a different set of random tags for each subtest to the _same_ set of random tags for all subtests. Not sure if that changes the nature of the test significantly.

Reply:
yeah, this would change the purpose of the test.
from earlier trying to understand if the test was passing without changing tags but never got moved back into the loop.